packages/cactus-plugin-htlc-eth-besu-erc20/README.md

```diff
@@ -0,0 +1,42 @@
+# `@hyperledger/cactus-plugin-htlc-eth-besu-erc20`
+
+Allows Cactus nodes to interact with HTLC contracts with ERC-20 Tokens
+
+## Summary
+
+  - [Getting Started](#getting-started)
+  - [Runing the tests](#running-the-tests)
+  - [Deployment](#deployment)
+  - [Built With](#built-with)
+  - [Contributing](#contributing)
+  - [License](#license)
+
+## Getting Started
+
+### Installing
+
+In a Cactus root directory, need execute:
+```
+   npm run run-ci
```

Comment:
@jordigiam Would recommend changing this to `npm run configure` which is much faster than than the full `run-ci` script. (About 1/5 the time I think)

Reply:
Done